tests/app/main/views/test_suppliers.py

```diff
@@ -111,55 +103,21 @@ def test_should_show_user_details_on_page(self, data_api_client):
 
         response = self.client.get('/admin/suppliers/users?supplier_id=1000')
 
-        self.assertEquals(200, response.status_code)
-        self.assertIn(
-            "Test User",
-            response.get_data(as_text=True)
-        )
-        self.assertIn(
-            "test.user@sme.com",
-            response.get_data(as_text=True)
-        )
-        self.assertIn(
-            "10:33:53",
-            response.get_data(as_text=True)
-        )
-        self.assertIn(
-            "23 July",
-            response.get_data(as_text=True)
-        )
-        self.assertIn(
-            "13:46:01",
-            response.get_data(as_text=True)
-        )
-        self.assertIn(
-            "29 June",
-            response.get_data(as_text=True)
-        )
-        self.assertIn(
-            "No",
-            response.get_data(as_text=True)
-        )
-
-        self.assertIn(
-            '<input type="submit" class="button-destructive"  value="Deactivate" />',
-            response.get_data(as_text=True)
-        )
-
-        self.assertIn(
-            '<form action="/admin/suppliers/users/999/deactivate" method="post">',
-            response.get_data(as_text=True)
-        )
+        assert response.status_code == 200
+        assert "Test User" in response.get_data(as_text=True)
```

Comment:
It might be worth setting a variable something like `page_as_text = response.get_data(as_text=True)` to avoid calling `get_data()` over and over.  For some very minimal value of "worth".